packages/synapse-react-client/src/components/QueryVisualizationWrapper.tsx

```diff
@@ -35,6 +35,10 @@ export type QueryVisualizationContextType = {
   /** React node to display in place of cards/table when there are no results. */
   NoContentPlaceholder: () => JSX.Element
   isRowSelectionVisible: boolean
+  isShowingExportToCavaticaModal: boolean
+  setIsShowingExportToCavaticaModal: React.Dispatch<
+    React.SetStateAction<boolean>
+  >
```

Comment:
Moved state to QueryVisualizationWrapper from TopLevelControls, since it's not necessarily true that you triggered the modal from TopLevelControls, it may be triggered from the RowSelection components.

Today, the RowSelectionControls is rendered by TopLevelControls, but I don't think it needs to be/should, so this change decouples them.

There's a lot of rearranging of state that I think we should do, but that could be a fairly large set of changes.